index.js

```diff
@@ -0,0 +1,85 @@
+const project = [
+  {
+    image: './image/pic.jpeg',
+    title: 'Multi-Post Stories Gain+Glory',
+    language: ['HTML5', 'HTML5', 'HTML5'],
+    description: 'lorem ipsum dolor sit amet consectetur adipisicing elit. Dolore distinctio aut illum officia unde, quod aliquam eos delectus aspernatur temporibus accusantium, sint quisquam sed quaerat provident tempora reprehenderit explicabo amet.',
+    liveLink: 'https://github.com/41hulk/micro-portfolio',
+    gitLink: 'https://github.com/41hulk/micro-portfolio',
+
+  },
+];
+
+const projectsDesktop = document.querySelector('.projects-desktop');
+
+for (let i = 0; i <= 5; i += 1) {
+  const card = document.createElement('div');
+  card.classList.add('project-container');
+
+  const img = document.createElement('img');
+  img.src = project[0].image;
+
+  const desc = document.createElement('div');
+  desc.classList.add('desc');
+
+  const title = document.createElement('h3');
+  title.innerHTML = project[0].title;
+
+  const languages = document.createElement('ul');
+
+  project[0].language.forEach((language) => {
+    const li = document.createElement('li');
+    li.innerHTML = language;
+    languages.appendChild(li);
+  });
+
+  const button = document.createElement('button');
+  const modal = document.querySelector('.modal-invisible');
+  button.setAttribute('type', 'button');
+  button.classList.add('project-button');
+  button.innerHTML = 'See project';
+
+  button.addEventListener('click', () => {
+    modal.style.display = 'block';
+    const content = `<div id="modal">
```

Comment:
- [ ] I recommend sticking the project design as much as possible because I think this is one of the crucial aspects of becoming a professional developer nowadays.  Please, take a look at the original and your design below and fix the differences 👇 
- [ ] Original mobile 
![Screenshot (150)_LI](https://user-images.githubusercontent.com/67087664/151544582-8351a9fb-853f-41ad-91b5-3d28d6917c51.jpg)
- [ ] Your mobile design 
![Screenshot (149)_LI](https://user-images.githubusercontent.com/67087664/151544653-5271ce6e-7b08-41a9-b398-03198eb6f475.jpg)
- [ ] Original desktop version
![Screenshot (148)_LI](https://user-images.githubusercontent.com/67087664/151544722-d8838be3-4117-40f1-870a-d0b9906ad8a6.jpg)
- [ ] Your desktop design 
![Screenshot (147)_LI](https://user-images.githubusercontent.com/67087664/151544764-70a46c66-ec69-4dec-9bb1-31d04a628fa0.jpg)
I have marked the differences with blue color, kindly use similar colors for your close buttons and make your `see live` and `see source` inputs larger in size.